### ISSUE TITLE
Typescript Element Type Problem

### ISSUE THREAD

**Issue by faddalibrahim** [faddalibrahim/freak-buzz#1]
Property `<html element>` does not exist on type JSX.IntrinsicElements

Examples
Property `'div'` does not exist on type `'JSX.IntrinsicElements'`
Property `'label'` does not exist on type `'JSX.IntrinsicElements'`

this [article](https://bobbyhadz.com/blog/react-property-does-not-exist-on-type-jsx-intrinsicelements#:~:text=The%20error%20%22Property%20does%20not,Copied!) explains why it happens but there is a mismatch

In summary it occurs for when a returned component's name starts with a small letter; for example returning `<myComponent/>` instead of `<MyComponent/>`

I'm assuming it happens because I'm returning `<div>`, `<label>` etc in component definitions. BUT these are native HTML elements not React Components.

Good thing is the code still compiles so we can move on!

PS: Will figure out a fix later, but if you find one, GREAT!
